### PR TITLE
Package updates

### PR DESCRIPTION
**Pull request by k80bowman**
Resolves all but one severe NPM vulnerabilities.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,24 +33,24 @@
...
-    "gulp": "^4.0.0",
+    "gulp": "^4.0.2",
     "gulp-clean": "^0.4.0",
     "gulp-rename": "^1.4.0",
     "gulp-string-replace": "^1.1.2",
     "lodash": "^4.17.21",
     "mini-css-extract-plugin": "^0.4.4",
     "mixin-deep": ">=1.3.2",
-    "node-sass": "^4.14.1",
+    "node-sass": "^6.0.1",
```
Comment (bkjohnson) [inline package.json:43]: I didn't find `node-sass` used anywhere, so it _might_ be safe to remove.

image

File: package.json [hunk 2]
```diff
@@ -33,24 +33,24 @@
...
     "raw-loader": "^0.5.1",
     "resolve-url-loader": "^3.0.0",
     "sass-loader": "^7.1.0",
     "set-value": ">=4.0.1",
     "style-loader": "^0.23.1",
     "tar": ">=6.1.9",
     "url-loader": "^1.1.2",
-    "webpack": "^4.43.0",
+    "webpack": "^5.64.4",
```
Comment (bkjohnson) [inline package.json:51]: I was still able to get the site to run with this major upgrade.

**Review by bkjohnson — APPROVED** [review]
I don't know if this is something we need to worry about - there were no major upgrades to `gulp`, but it isn't being found:

image

File: package.json (hunk 2 above)
Comment (k80bowman) [reply package.json:51]: yeah, I was amazed that all I had to do was make one change to the webpack config and everything was fine.

**Comment by k80bowman** [comment]
@bkjohnson that's interesting, my local build is able to find gulp...
image

**Comment by bkjohnson** [comment]
Maybe it's an npm bug. This is what I got when I did an earlier install before the build where `gulp` wasn't found:

image

So, maybe things didn't install right the first time. I just re-installed and did a build and gulp is working fine!

**Comment by k80bowman** [comment]
Oh interesting... look like I may need to update my node and npm versions.

**Comment by humancompanion-usds** [comment]
I think this was superseded by the work in issue #679

**Pull request closed without merging**